### PR TITLE
Imgcdngood

### PR DESCRIPTION
**Pull request by tqureshi-uog**
# Summary of changes
Gatsby Image CDN allows emulation of image cdn in local development. This results in huge speed increases when running `gatsby develop` since the process no longer needs to download every single image file in advance. This pull request implements the necessary updates to use Gatsby's Image CDN feature.

## Frontend
- All GraphQL image queries updated to use the new format (which will eventually be required regardless of whether a site uses Image CDN)
- All `file_media_image` calls updated to remove references to `localFile` and other incompatible code
- `gatsby-config.js` updated to set `skipFileDownloads` to `true`
- New class `img-dark` added to `global.css` for applying a brightness-reducing filter to images when needed (this is because Gatsby Image CDN GraphQL queries do not support duotone from `transformOptions`)

## Backend
- N/A

# Test Plan
- Verify that https://testchug.uoguelph.ca goes to the UoG homepage and _not_ the Gatsby index
- Go to https://testchug.uoguelph.ca/studentexperience and verify all images show up
- Go to https://testchug.uoguelph.ca/programs and choose any undergrad program
    - Verify the modal video plays
    - Verify all images show
    - View source for the page and verify the `og:image` tag shows an absolute URL starting with https://cdn.uoguelph.ca/ (the `og:image` tags are for URL preview cards and must use an absolute URL or else they won't work. If you have a Facebook account, you can also test the preview card on https://developers.facebook.com/tools/debug/)
    - Open browser inspector, view the Console tab, and verify there are no 404 errors for missing images or files (you should only see a few minified React errors which also appear on the live site and are an ongoing, separate, known issue)

### CODE REVIEW

**Comment by abanuog** [comment]
Went through the test plan and things worked as expected. Will review the files next

File: src/components/shared/storyImageCutout.js [hunk 1]
```diff
@@ -58,12 +58,12 @@ function StoryImageCutout (props) {
...
   let foreground = {
-    src: storyRelationships?.field_story_image?.relationships?.field_media_image.localFile,
+    src: storyRelationships?.field_story_image?.relationships?.field_media_image,
     alt: storyRelationships?.field_story_image?.field_media_image?.alt,
   }
   let background = {
-    src: storyRelationships?.field_story_image_bg?.relationships.field_media_image.localFile,
-    alt: storyRelationships?.field_story_image_bg?.field_media_image?.alt,
+    src: storyRelationships?.field_story_image_bg?.relationships.field_media_image,
+    alt: storyRelationships?.field_story_image?.field_media_image?.alt,
```
Comment (mmafe) [inline src/components/shared/storyImageCutout.js:66]:
```suggestion
    alt: storyRelationships?.field_story_image_bg?.field_media_image?.alt,
```

**Review by mmafe — CHANGES_REQUESTED** [review]
Update story_image_bg alt text

**Review by abanuog — APPROVED** [review]
Approved pending the change suggested by Miranda.

**Review by mmafe — APPROVED** [review]
Approved pending completed checks

**Pull request merged**